studymanager/src/main/java/io/redlink/more/studymanager/utils/SchedulerUtils.java

```diff
@@ -145,6 +145,7 @@ private static List<Instant> parseToInterventionSchedulesForScheduledTrigger(Tri
         if (CronExpression.isValidExpression(cronString)) {
             try {
                 CronExpression cronExpression = new CronExpression(cronString);
+                cronExpression.setTimeZone(TimeZone.getTimeZone(ZoneId.of("Europe/Vienna")));
```

Comment:
Good catch.

But we're using `ZoneId.systemDefault()` everywhere else in the code, so this should be aligned:

```suggestion
                cronExpression.setTimeZone(TimeZone.getTimeZone(ZoneId.systemDefault()));
```

We're aware that using `systemDefault()` can result in unexpected results, but that needs to be addressed systematically in a separate issue (TimeZone should become a dedicated setting for a study).